db/migrate/20191211120822_add_courses_recruitment_cycle_year.rb

```diff
@@ -0,0 +1,7 @@
+class AddCoursesRecruitmentCycleYear < ActiveRecord::Migration[6.0]
+  def change
+    add_column :courses, :recruitment_cycle_year, :integer
+    Course.update_all recruitment_cycle_year: 2020
+    change_column :courses, :recruitment_cycle_year, :integer, null: false
```

Comment:
FYI you can replace above 2 lines with `change_column_null(:courses, :recruitment_cycle_year, 2020)`

https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column_null

Reply:
Thanks, much tidier. More importantly this migration wasn't reversible - I guess we should be making that a requirement going forward.